### ISSUE TITLE
UpdatePolicy AutoScalingRollingUpdate does not accept a MinActiveInstancesPercent property.

### ISSUE THREAD

**Issue by sudoudaisuke** [aws-cloudformation/cfn-lint#3730]
### CloudFormation Lint Version

1.15.2

### What operating system are you using?

Ubuntu

### Describe the bug

UpdatePolicy AutoScalingRollingUpdate does not accept a MinActiveInstancesPercent /property.

```shell-session
$ cfn-lint test.yaml
E3016 Additional properties are not allowed ('MinActiveInstancesPercent' was unexpected)
test.yaml:11:9
```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html
https://github.com/aws-cloudformation/cfn-lint/blob/2e8bd3dd07aa23946f9496205b2bef78be725d76/src/cfnlint/data/schemas/other/resources/update_policy.json#L112C4-L112C28

### Expected behavior

success lint

### Reproduction template

```yaml
Resources:
  myLaunchTemplate:
    Type: AWS::EC2::LaunchTemplate
    Properties:
      LaunchTemplateData: {}
  myASG:
    Type: AWS::AutoScaling::AutoScalingGroup
    UpdatePolicy:
      AutoScalingRollingUpdate:
        MaxBatchSize: 2
        MinActiveInstancesPercent: 100
    Properties:
      MaxSize: "1"
      MinSize: "1"
      LaunchTemplate:
        LaunchTemplateId: !Ref myLaunchTemplate
        Version: !GetAtt myLaunchTemplate.LatestVersionNumber
```
